service/controller/v15/resource/deployment/master_deployment.go

```diff
@@ -321,6 +321,11 @@ func newMasterDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 									},
 								},
 							},
+							{
+								Name:            "shutdown-deferrer",
+								Image:           key.ShutdownDeferrerDocker,
+								ImagePullPolicy: apiv1.PullAlways,
```

Comment:
There shall be environment variables added here later to pass POD information (via k8s Downward API mechanism) to sidecar.